backend-tests/tests/test_inventory.py

```diff
@@ -14,31 +14,48 @@
 import pytest
 import random
 import time
+from datetime import datetime
 
 from api.client import ApiClient
-from common import mongo, clean_mongo
+from common import mongo, clean_mongo, clean_mongo_cls
 from infra.cli import CliUseradm, CliDeviceauth, CliTenantadm
 import api.deviceauth as deviceauth_v1
 import api.useradm as useradm
-import api.inventory as inventory
+import api.inventory as inventory_v1
+import api.inventory_v2 as inventory
```

Comment:
I can't see inventory_v2.py in api directory.

Reply:
fixed